lib/git_worktree.rb

```diff
@@ -53,18 +59,40 @@ def tags
     @repo.tags.each.collect(&:name)
   end
 
+  private def find_tag(name)
```

Comment:
For what it is worth... I definitely agree with rubocop here...  (read your comment, so don't change at this point).

Reply:
It's worth debating...I love this form (much easier to determine on git grep if something is private or not) 😄

Reply:
The following form:

```ruby
def find_tag(name)
  nil
end
private :find_tag
```

Also has the same effect you are talking about, but doesn't annoy `rubocop` (if I am not mistaken.

* * *

Side note:  Noticed the explicit `nil` in the body of this method is kind of excessive and not needed.

Reply:
> Also has the same effect you are talking about
> 
> Side note: Noticed the explicit nil in the body of this method is kind of excessive and not needed.

So, replace that nil with 20 lines of code and then `git grep "def find_tag"` and that's closer to what I meant with respect to git grep being nicer.

Reply:
Last two cents and I am going to resolve this (since we are in the weeds at this point):

my point was the following two "should" be equivalent in there accuracy:

```console
$ git grep 'private def find_tag'
$ git grep 'private :find_tag'
```

But honestly, I am rarely looking for "a private method specifically" in my search, and am usually unsure of the form, so I tend to default to:

```console
$ git grep "def.*find_tag"
```

Which will handle cases like this, but also if it happens to be a class method with:

```ruby
def self.find_tag
end
```

And that then handles the issue you are concerned about when greping and it turning you to the bottom of the method (since I am always searching for a `def ...`.